src/contracts/pods/EigenPod.sol

```diff
@@ -201,15 +217,13 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
         onlyWhenNotPaused(PAUSED_EIGENPODS_VERIFY_CREDENTIALS)
         // check that the provided `oracleTimestamp` is after the `mostRecentWithdrawalTimestamp`
         proofIsForValidTimestamp(oracleTimestamp)
-        //ensure that caller has restaking enabled by calling "activateRestaking()"
+        // ensure that caller has previously enabled restaking by calling `activateRestaking()`
```

Comment:
right now for a new EigenPod, won't a user have to:
1. deploy EigenPod
2. "enable restaking"
?

we should update the initializer for new EigenPods + the call made by the EigenPodManager, so that new pods automatically have "restaking enabled"

Reply:
Sorry I missed responding to this comment

I think the idea was that someone can deploy a pod, point their validator to it and then not be forced immediately into having to restake (which comes with proving, withdrawals both partial and full, etc.).,  When they're ready to turn it on, they can.

@gpsanant thoughts on this?

Reply:
not worth it for complexity imo, let them decide when they create the pod. agree w @ChaoticWalrus.

Reply:
Tracking this issue here @ChaoticWalrus : https://github.com/Layr-Labs/eignlayr-contracts/issues/965